views/index.hbs

```diff
@@ -0,0 +1,38 @@
+{{#if product}}
+<h1>{{type}} Selection</h1>
+{{#each products}}
+<div class="products">
+    <ul>
+        <li>
+            <h2>ID#: {{id}}
+                <a href='/products/{{id}}/edit'> <button class="edit">Edit: {{id}}</button></a>
+            </h2>
+            <h2>Name: <a href='/products/{{id}}'>{{name}}</h2> </a>
+            </a>
+            <h3>Price: ${{price}}</h3>
+            <h3>Stock: {{inventory}}</h3>
+        </li>
+    </ul>
+</div>
+{{/each}}
+{{else}}
```

Comment:
For all the hbs files below this, just make a different file for articles, and for products. Then just put them in different dirs. You'll need to update how you call render for those files but just think file directories.